_includes/releases/v23.1/v23.1.0-rc.1.md

```diff
@@ -0,0 +1,78 @@
+- Fixed a bug where a `RESTORE` operation with `skip_localities_check` could fail with errors if regions were missing on a cluster. [#101798][#101798]
+- Fixed a bug introduced in testing releases of v23.1 that could cause incorrect results for queries with `STRICT` user-defined functions. [#101951][#101951]
+- Fixed a rare bug that can cause keys to be unexpectedly deleted locally within a store by replica rebalancing during a write heavy workload. [#102166][#102166]
+- Fixed a bug where a failed or cancelled `IMPORT` operation could leave some of the imported rows behind after it was cancelled, if the writing processes were slow enough to continue writing after the cleanup process started. [#102246][#102246]
+- Fixed a bug in the behavior of the `enforce_home_region` [session variable](../v23.1/set-vars) that may have allowed a hash join to be favored over a lookup join, or failed to error out remote accesses done by uniqueness checks for mutations on R`EGIONAL BY ROW` tables. Also, static erroring of some locality-optimized lookup joins is now handled dynamically during query execution. [#102287][#102287]
+- Fixed a bug introduced in testing releases of v23.1 where a node could crash when evaluating a `COPY` command when the schema had `INT2` or `INT4` type. [#102307][#102307]
+- Fixes a bug where a backup of a key's revision history may not correctly be restored to the proper revision of the key if it is split across multiple sorted string tables. [#102342][#102342]
+- Fixed a bug introduced in testing releases of v23.1 where a user could be prevented from logging in or viewing or changing `GRANT`s if the cluster had a long period of inactivity. [#102488][#102488]
+- Fixed a bug where a node that transfers a lease away and misses a liveness heartbeat could then erroneously update the closed timestamp during the stasis period if its liveness. This could lead to closed timestamp invariant violation and could cause the node to crash. In extreme cases, this could lead to inconsistencies in read-only queries. [#102602][#102602]
+
+<h3 id="v23-1-0-rc-1-performance-improvements">Performance improvements</h3>
+
+- SQL Activity dashboards now default to using a table optimized with the top data for the most used cases. Otherwise, they use persisted data if available, and in-memory data otherwise. [#102310][#102310]
+
+<h3 id="v23-1-0-rc-1-miscellaneous">Miscellaneous</h3>
+
+- Two new statistics help to track the efficiency of snapshot transfers and to keep the number of failures due to system-level races as low as possible. `range.snapshots.recv-failed` shows the number of snapshot send attempts that are initiated but not accepted by the recipient. `range.snapshots.recv-unusable` shows the number of snapshots that were fully transmitted but not used. [#101835][#101835]
+
+<h4 id="v23-1-0-rc-1-changes-without-release-note-annotation">Changes without release note annotation</h4>
```

Comment:
This doesn't belong here. The note below needs to be moved to the appropriate location.